### ISSUE TITLE
Disable authentication

### ISSUE THREAD

**Issue by david-plugge** [louislam/uptime-kuma#1672]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

I just set up uptime kuma along with traefik + authelia (an sso authentication provider) and disabled authentication in uptime kuma. To do so i had to create an account.

### ✔️ Solution

Add an environment variable to disable the builtin authentication.

### ❓ Alternatives

Disable authentication in the web ui.

**Comment by jfcabral** [comment]
👍 I too would find this feature to be useful.  I believe there are use-cases where you simply want to show a dashboard, and make it available for a group of people, without the need to setup users and authentication.

**Comment by dwydler** [comment]
You have my vote too. I plan the same design for Uptime Kuma.

**Comment by allebone** [comment]
Just to clarify. Users like mine merely only need auth bypass for specific status pages. Not free reign to the dashboard.

**Comment by gaetanlord** [comment]
In the same topic, maybe a non login page displaying the status but restrict any changes. That would be a  great feature

**Comment by toxinliquid** [comment]
i would die for this feature

**Comment by flammable** [comment]
I might be misunderstanding, but I think some of these features already exist.

1. You can completely disable authentication (if you're using something like Authelia). This was [added in 1.0.8](https://github.com/louislam/uptime-kuma/releases/tag/1.0.8).
2. You can display a status page to users without them needing to authenticate (or have access to the dashboard). This was [added in 1.13.1](https://github.com/louislam/uptime-kuma/releases/tag/1.13.1).

**Comment by CommanderStorm** [comment]
@david-plugge what feature are you missing?
It seems to me that the feature you are referring to already exists. (as commented by @flammable)

If this issue is resolved, could you close this issue?
If it is not, could you comment further details what parts you are missing?

**Comment by david-plugge** [comment]
Yes, you can disable authentication once you created an account. But I was thinking about an environment variable like DISABLE_AUTH=1 to disable it so you don't need to create an account first.

**Comment by david-plugge** [comment]
Not yet, still waiting for it :)

**Comment by clitters** [comment]
This really would be useful. Im running my kuma inside a docker container, behind a reverse proxy that already handles my authentication. The container has no more exposed ports. Therefore i dont need/want any authentication ;)

**Comment by davidmreed** [comment]
@louislam I would like to implement this if you'd consider a small PR. It looks to me like a couple of lines' change in `server/auth.js` would do the trick.

**Comment by joshiparth1000** [comment]
at least auth should be disabled from /metrics to allow scraping without additional auth overhead
